README.md

```diff
@@ -113,6 +113,41 @@ Second, use the `HeraldPasswordResetForm` in place of django's built in `Passwor
 Third, you may want to customize the templates for the email. By default, herald will use the `registration/password_reset_email.html` that is provided by django for both the html and text versions of the email. But you can simply override `herald/html/password_reset.html` and/or `herald/text/password_reset.txt` to suit your needs.
 
 
+## Email Attachments
+
+To send attachments, assign a list of tuples to the attachments attribute of your EmailNotification instance, or override the get_attachments() method. The tuples should consist of the filename, the raw attachment data, and the mimetype.  It is up to you to get the attachment data.  Like this:
+
+    raw_data = get_pdf_data()
+
+    email.attachments = [
+       ('Report.pdf', raw_data, 'application/pdf'),
+       ('report.txt', 'text version of report', 'text/plain')
+    ]
+    email.send()
+
+You can also provide a MIMEBase object instead of a tuple.  See the documentation for attachments under EmailMessage Objects/attachments in the Django documentation.
+
+### Inline Attachments
+
+Sometimes you want to embed an image directly into the email content.  Do that by using a MIMEImage assigning a content id header to a MIMEImage, like this:
+
+    email = WelcomeEmail(user)
+    im = get_thumbnail(image_file.name, '600x600', quality=95)
+    my_image = MIMEImage(im.read()) # MIMEImage inherits from MIMEBase
+    my_image.add_header('Content-ID', '<{}>'.format(image_file.name))
+
+You can refer to these images in your html email templates using the Content ID (cid) like this:
+
+    <img src="cid:{{image_file.name}}" />
```

Comment:
@georgmzimmer in order to have `image_file` in the template, you need to add that to the email context right? Should we mention that in the above example?